helpinghand/apps/needs/serializers/v1_serializers.py

```diff
@@ -0,0 +1,6 @@
+# Local Django
+from .base_serializers import NeedListSerializer
+
+
+class NeedListSerializerV1(NeedListSerializer):
+    pass
```

Comment:
creator, supporters, categories için serializerlar NeedListSerializerdaki gibi eklenememiş.